compiler/luci/service/src/Nodes/CircleRange.cpp

```diff
@@ -84,6 +84,13 @@ loco::TensorShape Algorithm::visit(const luci::CircleRange *node)
   if (delta == 0)
     INTERNAL_EXN("Delta can not be zero");
 
+  /*
+   * Pre-condition
+   *    'limit - start' and 'delta' have the same sign.
+   *       c1. '(limit - start) >= 0' -> 'delta > 0'
+   *       c2. '(limit - start) < 0' -> 'delta < 0'
+   * https://github.com/tensorflow/tensorflow/blob/da82fa9/tensorflow/lite/kernels/range.cc#L49-L50
+   */
```

Comment:
if this is a precondition we can add `assert()` like
```suggestion
   */
   assert(start >= limit && delta < 0) || (start <= limit && delta > 0));
```

Reply:
if you add this `assert()`, it would be better to update commit title/message.

Reply:
I changed title to '[luci/service] Add Assert about Shape Inference Condition for Range' from '[luci/service] Add Comment about Shape Inference Condition for Range' :)